<launch>
  <!-- <arg name="model" default="$(env TURTLEBOT3_MODEL)" doc="model type [burger, waffle, waffle_pi]"/> -->
  <arg name="model" default="burger"/>
  <!-- No namespace here as we will share this description. 
       Access with slash at the beginning -->
  <!-- <param name="r_d" command="$(find xacro)/xacro $(find turtlebot3_description)/urdf/turtlebot3_$(arg model).urdf.xacro" /> -->

  <!-- BEGIN ROBOT 1-->
  <group ns="robot1">
    <param name="tf_prefix" value="robot1" />
    <include file="$(find multi_robot)/launch/TD_one_robot.launch" >
      <arg name="tbName" value="robot1"/>
      <arg name="x_pos" value="2.0"/>
      <arg name="y_pos" value="2.0"/>
      <arg name="z_pos" value="0.0"/>
    </include> 
  </group>

  <!-- BEGIN ROBOT 2-->
  <group ns="robot2">
    <param name="tf_prefix" value="robot2" />
    <include file="$(find multi_robot)/launch/TD_one_robot.launch" >
      <arg name="tbName" value="robot2"/>
      <arg name="x_pos" value="0.0"/>
      <arg name="y_pos" value="0.0"/>
      <arg name="z_pos" value="0.0"/>
    </include>
    
  </group>

  
  

</launch>